chainer/links/loss/hierarchical_softmax.py

```diff
@@ -349,6 +391,58 @@ def create_huffman_tree(word_counts):
 
         return q.get()[2]
 
+    def argmax(self, x):
+        """Argmax an example for a given input from the tree.
+
+        Args:
+            x (~chainer.Variable): Input variable for sample word ids.
+
+        Returns:
+            array: Array of word indexes in a binary tree ``self.tree``.
+
+        .. admonition:: Example
+
+            Let an input vector ``x`` be:
+
+            >>> word_cnts = {0: 8, 1: 5, 2: 6, 3: 4, 4: 10, 5: 1, 6: 32}
+            >>> tree = BinaryHierarchicalSoftmax.create_huffman_tree(word_cnts)
+            >>> hsm = BinaryHierarchicalSoftmax(3, tree)
+            >>> x = np.array([[0.2, 0.2, 0.3], [0.1, 0.3, 0.1]], dtype='f')
+            >>> hsm.argmax(Variable(x))
+            [0, 3]
+        """
+        if len(self.tree) == 0:
```

Comment:
In this case, HSM link itself does not work, right? If so, please check it in `__init__`.